api/graphql/makeModels.js

```diff
@@ -457,23 +482,55 @@ export default function makeModels (userId, isAdmin, apiClient) {
     CustomView: {
       model: CustomView,
       attributes: [
+        'active_posts_only',
+        'collection_id',
+        'default_sort',
+        'default_view_mode',
+        'external_link',
         'group_id',
-        'is_active',
-        'search_text',
         'icon',
+        'is_active',
         'name',
-        'external_link',
-        'view_mode',
-        'active_posts_only',
+        'order',
         'post_types',
-        'order'
+        'type',
+        'search_text',
       ],
       relations: [
+        'collection',
         'group',
         { tags: { alias: 'topics' } }
       ]
     },
 
+    Collection: {
+      model: Collection,
+      attributes: [
+        'created_at',
+        'name',
+        'updated_at'
+      ],
+      relations: [
+        'group',
+        { linkedPosts: {querySet: true} },
+        { posts: {querySet: true} },
```

Comment:
whats the different between posts and linkedPosts, for my own understanding?

Reply:
linkedPosts is the join table entities